tests/mocha/workspace_svg_test.js

```diff
@@ -93,7 +93,7 @@ suite('WorkspaceSvg', function() {
         this.workspace.getTopBlocks()[0].getInput('NAME').connection;
     inputConnection.connect(block.outputConnection);
     chai.assert.exists(block.getSvgRoot());
```

Comment:
Can you update the assertion here as well? or remove it.

Reply:
Done!